config/src/main/java/com/alibaba/nacos/config/server/model/ConfigHistoryInfoPair.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.config.server.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Objects;
+
+/**
+ * ConfigHistoryInfoPair.
+ *
+ * @author dirtybit
+ */
+public class ConfigHistoryInfoPair implements Serializable {
```

Comment:
rename to ConfigHistoryDetailInfo

Reply:
I'm sorry I just saw your review, and I saw that you have already submitted a new PR to modify it, so I won't do it. If there are any problems or adjustments in this part later, I can continue to optimize it. In short, thank you for pointing out many irregularities in my code, which benefited me a lot.